### PR TITLE
Remove duped code

### PR DESCRIPTION
**Pull request by vext01**
This removed the last of the duplicated libkrun code from VMs. Rejoice!

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -241,7 +241,7 @@ build_v8() {
     #
     # V8 also mistakes our compiler for clang for some reason, hence
     # setting GYP_DEFINES.
-    env GYP_DEFINES="clang=0" CC=${OUR_CC} CXX=${OUR_CXX} ${GMAKE} native
+    env GYP_DEFINES="clang=0" CC=${OUR_CC} CXX=${OUR_CXX} ${GMAKE} native V=1
```
Comment (ltratt) [inline build.sh:244]: This feels like it's not supposed to be part of this PR?

Comment (vext01) [inline build.sh:244]: You can have it in a PR on it's own if you want. I think it's OK as long as we don't squash that commit.

Comment (ltratt) [inline build.sh:244]: OK.

**Comment by ltratt** [comment]
Do you want to squash or is it good as-is?

**Comment by vext01** [comment]
I'm going to squash away the accidental commit.

**Comment by vext01** [comment]
Here we are

**Pull request merged**